### DISCUSSION TITLE
What benefit can we get from hackathons or workshops?

### DISCUSSION THREAD

**Discussion by epijim** [phuse-org/devops#7] [Ideas]
Hackathon: building new tools

Workshops: helping users apply these tools (like what was done at R/Pharma)

Venues: Phuse CSS, PHUSE US, R/pharma workshop, etc

**Comment by bms63** [comment]
I am weary of Hackathons as they are a lot of work.  

From my vantage point as `cicdnewbguy` that the CI space is pretty well-figured out for R packages.  Is the hackathon looking at going beyond R packages?  I think that would be very interesting, but being a judge in that type of hackathon would be tough for me.  Still open to it.

Perhaps bi-annual workshops showcasing GitHub Actions from the WG would be better than a hackathon.
